markdown/developers/gitpod/git_in_gitpod.md

```diff
@@ -0,0 +1,11 @@
+## Making changes to a repository and commiting to git.
+
+Gitpod environments are a handy place to try out Nextflow and nf-core tools, test new features and make suggested changes to the underlying code. 
+
+Once you have made changes, you can push these to a new branch for review by the nf core team. 
+
+First, make sure your edited files are saved in your Gitpod IDE. Then click the Git checkout branch option at the bottom left of the window. Here will pop up an option to "Create a new branch...". Name a new branch and now you are operating on a branch of the main repository.
+
+To stage files to this new branch, click on the **Source Control** button (on the left hand panel).This shows all the changes to files and you can stage all changes by clicking the + button when you hover over the file name.
+
+Finally, type in a commit message and click the tick button at the top of the source window. Then click sync changes. You should have now pushed the new files to your new branch. 
```

Comment:
It might also be useful to demonstrate how this can be done from the command-line too, e.g
```
git checkout -b my_awesome_suggestion
#edit file
git add <file>
git commit -m "My informative commit message"
```

Reply:
Done, added a new header for option to use command line (very brief, could be expanded)